msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -1001,6 +1002,23 @@ The `extensions.runtimes` array configures the sets of runtimes and actions that
 
 To use `extensions.runtimes`, see [create add-in commands](/office/dev/add-ins/develop/create-addin-commands-unified-manifest), [configure the runtime for a task pane](/office/dev/add-ins/develop/create-addin-commands-unified-manifest#configure-the-runtime-for-the-task-pane-command), and [configure the runtime for the function command](/office/dev/add-ins/develop/create-addin-commands-unified-manifest#configure-the-runtime-for-the-function-command).
 
+### extensions.contentRuntimes
+
+The `extensions.contentRuntimes` array configures pages of content that are embedded in an Excel or PowerPoint document. 
+
+|Name| Type| Maximum size | Required | Description|
+|---|---|---|---|---|
+|`id`| String | 64 characters | ✔️ | Specifies the ID for runtime. Must be unique across all `runtimes` and `contentRuntimes` objects. |
+|`code`| Object | | ✔️ | Specifies the location of content for the runtime. |
+|`code.page`| URL | | ✔️ | The full URL of the page that contains the content that is embedded in the document. |
+|`requestedWidth`| Number | | | The desired width between 32 and 1000 pixels. If this property isn't used, the Office application determines the width.|
+|`requestedHeight`| Number | | | The desired height between 32 and 1000 pixels. If this property isn't used, the Office application determines the height.|
+|`disableSnapshot` | boolean | | | Specifies whether a snapshot image of your content add-in is saved with the host document. Default value is `false`, so a snapshot is saved. Set to `true` to prevent a snapshot from being saved.|
+|`requirements`| Object | | | Specifies the scopes, formFactors, and Office JavaScript Library requirement sets that must be supported on the Office client in order for the content to be embedded in the document. For more information, see [Specify Office Add-in requirements in the unified manifest for Microsoft 365](/office/dev/add-ins/develop/requirements-property-unified-manifest).|
+|`requirements.capabilities`| Array | | | Don't use when the grandparent is a content runtime object. |
+|`requirements.scopes`| Array of enums | 1 | | Identifies the scopes in which the add-in can run and defines the Microsoft 365 applications in which the extension can run. For example, `workbook` (Excel). <br>Supported values: `workbook`, `presentation` |
+|`requirements.formFactors`| Array of enums | | | Don't use when the grandparent is a content runtime object. |
```

Comment:
> Please check whether this property can be removed since we are adding it under contentRuntimes.

It cannot be removed because the "contentRuntimes.requirements" property uses the standard data type of "requirements" properties, so "capabilities" will appear in Intellisense even when the grandparent is a content runtimes object.